### ISSUE TITLE
Python if-else spacing

### ISSUE THREAD

**Issue by MeWhenI** [jared-hughes/polygolf#46]
Compiling
```
if (1==1) { println 1; } { println 2; };
```
to Python produces
```Python
if 1==1:print(1)else:print(2)
```
This is a syntax error, it should produce
```Python
if 1==1:print(1)
else:print(2)
```

**Comment by MichalMarsalek** [comment]
Fixed by #47 .

**Comment by MichalMarsalek** [comment]
Fixed by #47 .
